api/auth/serialization.js

```diff
@@ -25,6 +25,9 @@ module.exports.deserializeUser = async (
       done(null, {
         username: user.get('email'),
         id: user.get('id'),
+        name: user.get('name'),
+        phone: user.get('phone'),
+        position: user.get('position'),
```

Comment:
Return these as part of the user object, so when we add a UI for users to modify their own accounts, they'll be able to see what values are here already.  Because...  I don't know why these weren't already returned.  🤷‍♂️